temporal-sdk/src/main/java/io/temporal/internal/replay/WorkflowRunTaskHandler.java

```diff
@@ -35,10 +35,11 @@ public interface WorkflowRunTaskHandler {
    *
    * @param workflowTask task to handle
    * @return an object that can be used to build workflow task completion or failure response
+   * @throws Throwable if processing experienced unrecoverable unexpected issues
```

Comment:
Should this say "unrecoverable"? Since the idea is to let it try again with WFT failure? Unrecoverable for some smaller scope, but recoverable in the bigger sense.

Reply:
Yeah, it may be unclear with this wording. I actually mean "Unrecoverable on this worker in the context of this Workflow Task". Because you are totally correct, it doesn't mean that it's unrecoverable for the workflow, hence WFT failure. I will add a clarification.